src/Controls/src/Core/DragAndDrop/DragGestureRecognizer.cs

```diff
@@ -90,27 +103,26 @@ internal void SendDropCompleted(DropCompletedEventArgs args)
 			DropCompleted?.Invoke(this, args);
 		}
 
-		internal DragStartingEventArgs SendDragStarting(IView element)
+		// TODO: JD - Maybe mark this as obsolete?
+		internal DragStartingEventArgs SendDragStarting(IView element) => SendDragStarting(element, Point.Zero);
+		
+		internal DragStartingEventArgs SendDragStarting(IView element, Point position)
```

Comment:
There is no need to do this if you do the following trick:
```suggestion
		internal DragStartingEventArgs SendDragStarting(IView element, Point position = Point.Zero)
```
Problem solved, no need to Obsolete and the API is the same. Since the method is internal, we are happy :)

Reply:
We can't use Point.Zero as a default param because  it triggers [CS1736 "default parameter must be compile time constant"](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1736?f1url=%3FappId%3Droslyn%26k%3Dk(CS1736))

I've opted to just pass a `new Point()`, which is basically the same thing as Point.Zero